rest-service/setup.py

```diff
@@ -38,7 +38,6 @@
         'requests==2.7.0',
         'gunicorn==18.0',
         'PyYAML==3.10',
-        'elasticsearch==1.6.0',
```

Comment:
You should probably leave this line..
In order to be more explicit and safe perhaps the modified client version should be changed to something like 1.6.0.1 (because elasticsearch is still required by this project).

The github dependency of the modified es client should probably also be added to dev-requirements.txt (for unit tests and development).